crypto/cast/cast_local.h

```diff
@@ -7,6 +7,13 @@
  * https://www.openssl.org/source/license.html
  */
 
+# include "openssl/opensslconf.h"
+
+#ifdef OPENSSL_FIPS
```

Comment:
Is this extra code? I don't see anything in this file that would require fips.h/err.h

Reply:
Nope. Removing it gives:
```
cast/c_cfb64.d.tmp -MT crypto/cast/c_cfb64.o -c -o crypto/cast/c_cfb64.o crypto/cast/c_cfb64.c
crypto/cast/c_cfb64.c: In function 'CAST_cfb64_encrypt':
crypto/cast/c_cfb64.c:29:9: warning: implicit declaration of function 'FIPS_mode' [-Wimplicit-function-declaration]
   29 |     if (FIPS_mode()) {
      |         ^~~~~~~~~
crypto/cast/c_cfb64.c:30:9: warning: implicit declaration of function 'FIPSerr' [-Wimplicit-function-declaration]
   30 |         FIPSerr(ERR_LIB_FIPS, FIPS_R_NON_FIPS_METHOD);
etc....
```

Reply:
this is a local_include for most of the `crypto/cast/*.c`